auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -140,21 +140,30 @@
         <template #item-slot-Status="{ item }">
           <span>{{ status(item) }}</span>
           <!-- Future Effective Icon for PAID_FE status -->
-          <template v-if="status(item) === BusinessState.PAID_FE">
-            <v-tooltip bottom>
-              <template #activator="{ on, attrs }">
-                <v-icon
-                  class="ml-1"
-                  color="#F8661A"
-                  v-bind="attrs"
-                  v-on="on"
-                >
-                  mdi-information-outline
-                </v-icon>
-              </template>
-              <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
-            </v-tooltip>
-          </template>
+          <IconTooltip
+            v-if="status(item) === BusinessState.PAID_FE"
+            icon="mdi-information-outline"
+            maxWidth="300px"
+            :iconStyling="{'font-size': '1.5em', 'margin-left': '4px'}"
+            :location="{top: true}"
+          >
+            <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
+          </IconTooltip>
+
+          <!-- Alert Icon for CHANGE_REQUESTED status -->
+          <IconTooltip
+            v-if="status(item) === BusinessState.CHANGE_REQUESTED"
+            icon="mdi-alert"
+            maxWidth="300px"
+            :colour="'#F8661A'"
+            :iconStyling="{'font-size': '1.5em', 'margin-left': '4px'}"
+            :location="{top: true}"
+          >
+            <div>
+              <strong>Alert:</strong><br>
+              <span> This application requires you to make changes before resubmitting.</span>
+            </div>
+          </IconTooltip>
```

Comment:
Change Requested icon: Added line "Alert:..."
A single line, so no pointer

Reply:
Looks good:

![image](https://github.com/user-attachments/assets/5de108fc-3422-4049-b02a-de836ce448ce)

Reply:
What do you mean by "pointer"?

I was thinking about that little triangle at the bottom of the tooltip that points to the icon. And it's there, as it should be.